configurations/br-prod.ts

```diff
@@ -0,0 +1,58 @@
+import { PreEngagementConfig, Translations, Configuration, MapHelplineLanguage } from './types';
+
+const accountSid = 'AC98b66f5541f81de7050c7254cf5c96c9';
+const flexFlowSid = 'FO9dc6203414bd64ab7c39237ea3b8fe53';
+const defaultLanguage = 'en-US';
+const captureIp = true;
+
+const translations: Translations = {
+    'en-US': {
```

Comment:
Placeholder values here for now, just so we can see that webchat is working in the new environment